src/domain/Player.java

```diff
@@ -19,6 +24,17 @@ public String getName() {
 	public void giveCard(Card c) {
 		cards.add(c);
 	}
+	public List<Card> Getcard() {
```

Comment:
Het `makeMove` gedeelte hoort thuis in `Game`. Dus deze methode mag hier weg denk ik.